docs/03-dev-howtos/05-interactives.md

```diff
@@ -34,57 +58,42 @@ define(['your/dependencies'], function (dependency) {
     });
 ```
 
-## Embedding the interactive in a page
+## Interactive content atoms
+
+Interactive content atoms are the recommended way to create interactive content going forward. They are treated as first
```

Comment:
should this be at the top maybe? with a mention of deprecating the other two methods?